pymilvus/orm/connections.py

```diff
@@ -214,7 +216,7 @@ def remove_connection(self, alias: str):
         self.disconnect(alias)
         self._alias.pop(alias, None)
 
-    def connect(self, alias=Config.MILVUS_CONN_ALIAS, user="", password="", **kwargs):
+    def connect(self, alias=Config.MILVUS_CONN_ALIAS, user="", password="", db_name="default", **kwargs):
```

Comment:
It is recommended to fill in empty "or None here, and convert it to default at the server.